source/Outfit.cpp

```diff
@@ -30,7 +30,9 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 using namespace std;
 
 namespace {
-	const double EPS = 0.0000000001;
+	// Attributes are stored as integers but used as doubles. This is the factor by which to convert
+	// attribute values from doubles to integers and back.
+	constexpr int ATTRIBUTE_PRECISION = 10000;
```

Comment:
```suggestion
	// Attributes are stored as integers but used as doubles. This is the factor by which to convert
	// attribute values from doubles to integers and back. Using 0xffff (65535), because powers of
	// two are faster to multiply and divide than powers of ten.
	constexpr int ATTRIBUTE_PRECISION = 0xffff;
```